profiles/kentik_snmp/broadforward/diameter.yml

```diff
@@ -0,0 +1,168 @@
+        name: dmtrPeerTxCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1009
+        name: dmtrPeerSendQueueSize
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1011
+        name: dmtrPeerTxFailedCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1012
+        name: dmtrPeerRxResult1xxxCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1013
+        name: dmtrPeerRxResult2xxxCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1014
+        name: dmtrPeerRxResult3xxxCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1015
+        name: dmtrPeerRxResult4xxxCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1016
+        name: dmtrPeerRxResult5xxxCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1017
+        name: dmtrPeerRxResultOtherCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1018
+        name: dmtrPeerRxBytesCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1019
+        name: dmtrPeerTxBytesCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1020
+        name: dmtrPeerTimeoutCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1021
+        name: dmtrPeerRxRequestCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1022
+        name: dmtrPeerRxResponseCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1023
+        name: dmtrPeerTxRequestCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1024
+        name: dmtrPeerTxResponseCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1025
+        name: dmtrPeerTxPending
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1030
+        name: dmtrPeerRxInvalidCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1031
+        name: dmtrPeerRxInvOrigCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1040
+        name: dmtrPeerRxReqInvalidCount
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1120
+        name: dmtrPeerSctpRxSacks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1121
+        name: dmtrPeerSctpTxSacks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1122
+        name: dmtrPeerSctpRxPackets
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1123
+        name: dmtrPeerSctpTxPackets
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1124
+        name: dmtrPeerSctpRxUnordDataChunks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1125
+        name: dmtrPeerSctpTxUnordDataChunks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1126
+        name: dmtrPeerSctpRxOrdDataChunks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1127
+        name: dmtrPeerSctpTxOrdDataChunks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1128
+        name: dmtrPeerSctpRxControlChunks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1129
+        name: dmtrPeerSctpTxControlChunks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1140
+        name: dmtrPeerSctpRetransChunks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1141
+        name: dmtrPeerSctpOutOfSeqTSN
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1142
+        name: dmtrPeerSctpRxDups
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1143
+        name: dmtrPeerSctpRxGapAcks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.
+        name: dmtrPeerSctpMaxRTO
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.
+        name: dmtrPeerSctpMaxRTO
```

Comment:
the root OID on these seems to be wrong; should it be `1.3.6.1.4.1.39216.1.50.1.2.1.*` ? 

also; a few spot-checked as string values that need to be moved to `metric_tags`

Reply:
fixed